flask_appbuilder/models/mixins.py

```diff
@@ -48,17 +48,15 @@ class AuditMixin(object):
 
     @declared_attr
     def created_by_fk(cls):
-        return Column(Integer, ForeignKey('ab_user.id'),
-                      default=cls.get_user_id, nullable=False)
+        return Column(Integer,  default=cls.get_user_email, nullable=False)
 
     @declared_attr
     def created_by(cls):
-        return relationship("User", primaryjoin='%s.created_by_fk == User.id' % cls.__name__, enable_typechecks=False)
+        return relationship("User", primaryjoin='%s.changed_by_fk == User.id' % cls.__name__, enable_typechecks=False)
```

Comment:
Why it was change from created_by_fk to changed_by_fk in created_by method kub?

Reply:
mistake ka, will take it back to created_by_fk